media/js/devreg/login.js

```diff
@@ -160,20 +160,41 @@ define('login', ['notification', 'storage'], function(notification, storage) {
         }
     }
 
-    function init_persona() {
-        $('.browserid').css('cursor', 'pointer');
+    function init_native_fxa() {
         var user = z.body.data('user');
         var email = user ? user.email : '';
         console.log('detected user', email);
-        navigator.id.watch({
-            loggedInUser: email,
-            onlogin: function(assertion) {
-                if (!email) {
-                    gotVerifiedEmail(assertion);
+        var packaged_origin = "app://packaged." + window.location.host;
```

Comment:
I don't believe any of the non-fireplace pages (e.g. dev hub; reviewer tools) run directly in the packaged app, but instead launch out into a browser instance.